src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+import usersFromServer from '../../api/users';
+import { User } from '../../types/User';
+import { getUserById } from '../../services/user';
+
+type Props = {
+  onSubmit: (todo: Todo) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ onSubmit }) => {
+  const [title, setTitile] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
```

Comment:
u can use enum for ur errors. try it. for all such cases
`ErrorType.Default = ''`
```suggestion
  const [hasTitleError, setHasTitleError] = useState(ErrorType.Default);
```